src/rocksdb_options.rs

```diff
@@ -23,37 +23,91 @@ use merge_operator::{self, MergeOperatorCallback, MergeOperands, full_merge_call
               partial_merge_callback};
 use comparator::{self, ComparatorCallback, compare_callback};
 
-#[derive(Copy, Clone)]
-pub struct RocksDBOptions {
+pub struct BlockBasedOptions {
+    inner: rocksdb_ffi::RocksDBBlockBasedTableOptions,
+}
+
+pub struct Options {
```

Comment:
I'm totally on-board for simplifying these variable names, thanks for beginning this process.